src/tree_builder/rules.rs

```diff
@@ -770,7 +770,7 @@ impl<Handle, Sink> TreeBuilderStep
                     Done
                 }
 
-                // FIXME: This should be unreachable, but match_token! requires a
+                // FIXME: This should be unreachable, but match_token requires a
```

Comment:
Yes, the parser is that stupid. It would find `match_token!` in a comment and try to parse it as a macro invocation.